django_celery_beat/migrations/0005_auto_20170425_1819.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
this migration file needed to be deleted and regenerated after pulling from master